Detectors/ITSMFT/ITS/RawPixelReaderWorkflow/src/RawPixelGetterSpec.cxx

```diff
@@ -0,0 +1,68 @@
+#include <vector>
+
+#include "Framework/ControlService.h"
+#include "ITSRawWorkflow/RawPixelGetterSpec.h"
+#include "ITSMFTBase/Digit.h"
+#include "ITSMFTReconstruction/ChipMappingITS.h"
+#include "DataFormatsITSMFT/CompCluster.h"
+#include "DataFormatsITSMFT/Cluster.h"
+#include "SimulationDataFormat/MCCompLabel.h"
+#include "SimulationDataFormat/MCTruthContainer.h"
+#include "DataFormatsITSMFT/ROFRecord.h"
+#include "DataFormatsParameters/GRPObject.h"
+#include "ITSMFTReconstruction/DigitPixelReader.h"
+#include "DetectorsBase/GeometryManager.h"
+#include "ITSBase/GeometryTGeo.h"
+
+using namespace o2::framework;
+
+namespace o2
+{
+namespace its
+{
+
+void RawPixelGetter::init(InitContext& ic)
+{
+  LOG(DEBUG) << "Now Working on the GETTER BROS";
+
+  o2::base::GeometryManager::loadGeometry(); // for generating full clusters
```

Comment:
Why do you load geometry if you never use it?

Reply:
Suggestions taken! Fixed!